gradle.properties

```diff
@@ -0,0 +1,3 @@
+org.gradle.caching=true
+org.gradle.parallel=true
+org.gradle.vfs.watch=true
```

Comment:
I'm not convinced if we should enable it at the project level. It's rather user environment specific decision.

Reply:
It's enabled by default in Gradle 7 so I think that's ok.